.github/workflows/patch-release-build.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         uses: aws-actions/configure-aws-credentials@v1
         with:
           role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_ARN_RELEASE }}
-          aws-region: us-west-2
+          aws-region: ${{ env.AWS_DEFAULT_REGION }}
```

Comment:
oh, are we running tests in us-east-1 now?

Reply:
this image is not for tests. it is the image containing the ADOT Java Agent used by the operator. It has to use us-east-1 because we are publishing to a public ecr repository.

Reply:
Thanks for clarifying